botocove/cove_runner.py

```diff
@@ -78,6 +79,12 @@ def cove_thread(
             return cove_session.format_cove_result(result)
 
         except Exception as e:
+
+            if isinstance(e, EndpointConnectionError):
```

Comment:
this is a strange enough thing to look for that it seems worth commenting exactly why we're checking

Reply:
I thought I had commented it somewhere, but I seem to have lost that text before pushing. I agree. I'll add something.